.zenodo.json

```diff
@@ -84,6 +84,10 @@
       "orcid": "0000-0002-8508-9088"
     },
     {
+      "name": "Cieslak, Matthew",
+      "affiliation": "Department of Neuropsychiatry, University of Pennsylvania",
+      "orcid": "0000-0002-1931-4734"
+    },
       "name": "Liem, Franz",
```

Comment:
```suggestion
    {
      "name": "Liem, Franz",
```

Reply:
Just realizing, your entry should also be moved down to immediately before Russ. At least for now, we're using the order of first contribution, with Russ and Chris as co-last authors.

Reply:
Aha, I was wondering how this worked. In my copy there is someone between Chris and Russ, should I move Romain elsewhere?

Reply:
I just fixed it on master.

Reply:
This still needs updating. Please move your entry to just before Russ.